pbd_msgs/PbdAttributedPointCloud point_cloud
---
bool is_valid